src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FallThroughCheck.java

```diff
@@ -102,17 +64,118 @@
  * To configure the check:
  * </p>
  * <pre>
- * &lt;module name=&quot;FallThrough&quot;/&gt;
+ * &lt;module name="FallThrough"/&gt;
+ * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * switch (i) {
+ *   case 1:
+ *     i++;
+ *   case 2: // violation, case 1 contains code but lacks break, return, throw or continue statement
+ *     i++;
+ *     break;
+ *   case 3: // OK
+ *     i++;
+ *     return;
+ *   case 4: // OK
+ *     i++;
+ *     throw new Exception();
+ *   case 5: // OK
+ *     i++;
+ *     continue;
+ *   case 6: // OK
+ *   case 7: // OK, case 6 does not contain code
+ *     i++;
+ *   // OK, by default the last case is not checked
+ * }
+ * </pre>
+ * <p>
+ * Unreachable code still causes violation:
```

Comment:
please remove this section and its example, please address only properties and things mentioned in description.

Reply:
There is a `Note: The check assumes that there is no unreachable code in the case` in the description.
Should I still remove this section?

Reply:
I think yes, it is a rare case.

Reply:
fixed